contracts/test/vault/oeth-vault.fork-test.js

```diff
@@ -61,40 +61,37 @@ describe("ForkTest: OETH Vault", function () {
       oethWhaleSigner = await impersonateAndFund(oethWhaleAddress);
     });
 
-    it("should mint using each asset", async () => {
-      const { oethVault, oethOracleRouter, weth, frxETH, stETH, reth, josh } =
```

Comment:
I think we should add another test where minting with each of the LSTs reverts

Reply:
I think this is covered in the next test: 
<img width="620" alt="image" src="https://github.com/OriginProtocol/origin-dollar/assets/837/9166d0f3-8cf0-4898-8c9f-9f9f42583abf">